LittleDatabaseApp/Program.cs

```diff
@@ -0,0 +1,194 @@
+using System.Data.Entity;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LittleDatabaseApp
+{
+    class Program
+    {
+        public static bool exitApp;
+
+        public static List<Human> humans = new List<Human>();
+        public static List<Android> androids = new List<Android>();
+
+        static void Main(string[] args)
+        {
+            Database.SetInitializer(new DataInitializer());
+
+            while (!exitApp)
+            {
+                DisplayMenu();
+            }
+        }
+
+        private static void DisplayMenu()
+        {
+            Console.WriteLine($"Choose option\n1) Add Person\n2) Show peoples in memory\n3) Add people to database\n4) Show database records\n5) Exit App");
+
+            string userInput = Console.ReadLine();
+
+            if (userInput == "5")
+            {
+                exitApp = true;
+                return;
+            }
+
+            RecognizeUserInput(userInput);
+
+            Console.ReadLine();
+            Console.Clear();
+        }
+
+        private static void RecognizeUserInput(string userInput)
+        {
+            MenuOptions MenuChoosen = new MenuOptions();
+
+            if (!int.TryParse(userInput, out int userNumberInput))
+            {
+                    Console.WriteLine("Can't recognize answer");
+            }
+
+            if (userNumberInput == 1 || userNumberInput == 2 || userNumberInput == 3 || userNumberInput == 4)
+            {
+                MenuChoosen = (MenuOptions)userNumberInput - 1;
+                ChooseRightMenu(MenuChoosen);
+            }
+            else
+            {
+                Console.WriteLine("Choose the correct number");
+            }
+        }
+
+        private static void ChooseRightMenu(MenuOptions menu)
+        {
+            switch (menu)
+            {
+                case MenuOptions.addHuman:
+                    Console.Clear();
+                    AddHumanMenu();
+                    break;
+                case MenuOptions.memory:
+                    Console.Clear();
+                    PeopleInMemoryMenu();
+                    break;
+                case MenuOptions.addPeopleToDatabase:
+                    Console.Clear();
+                    AddPeopleToDatabaseMenu();
+                    break;
+                case MenuOptions.database:
+                    Console.Clear();
+                    ShowDatabaseMenu();
+                    break;
+                default:
+                    break;
+            }
+        }
+
+        private static void AddHumanMenu()
+        {
+            string firstName;
+            string lastName;
+
+            Console.Write("First Name: ");
+            firstName = Console.ReadLine();
+
+            Console.Write("Last Name: ");
+            lastName = Console.ReadLine();
+
+            try
+            {
+                Human person = new Human() { FirstName = firstName, LastName = lastName};
```

Comment:
It would be cleaner if you would create the `Human` instance in that way:
```C#
var human = new Human
{
    FirstName = firstName,
    LastName = lastName
};
```
And then a second good practice to cleaner code would be adding one line break before and after the statement, I wrote above.